sections/builder.js

```diff
@@ -2282,5 +2282,75 @@ export default [
         content: "Calling `queryContext` with no arguments will return any context configured for the query builder instance."
       }
     ]
-  }
+  },
+  {
+    type: "heading",
+    size: "md",
+    content: "Extending Query Builder",
+    href: "Builder-extending"
+  },
+  {
+    type: "text",
+    content: [
+      "**Important:** this feature is experimental and it's API may change in the future.",
+      "It allows to add custom function the the Query Builder. It should be invoked only once in the app.",
```

Comment:
`It should be invoked only once in the app` -> this sounds confusing. Does it mean you can only add one method, or that after you invoke it once, it applies to all QueryBuilder instances?

Reply:
It means that in-order to add new method you need to add each method only once.

Reply:
I will remove this sentence, it's just confusing :], If some one will add the same method twice it will throw exception that this method is already exists.